src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -1656,6 +1657,24 @@ private static void ValidateWin32Settings(string win32ResourceFile, string win32
             }
         }
 
+        internal static IEnumerable<InstrumentationKind> ParseInstrumentationKinds(string value, IList<Diagnostic> diagnostics)
```

Comment:
Not part of this change but it looks like `ParseAnalyzers` and `ParseAssemblyReferences` can be `static`. Same for `ParseAnalyzers` in VB.